### ISSUE TITLE
🐛 BUG: Astro.play cant import internal module

### ISSUE THREAD

**Issue by X7md** [withastro/astro-repl#25]

**Comment by okikio** [comment]
I don't think this is possible in the Astro repl right now, but I'll see if I can add support for it.

**Comment by okikio** [comment]
The changes required to make `astro/components` work are larger than what I initially thought, it would require setting up the repo to accept external astro files, which would require an infrastructure upgrade. It may take longer than I initially thought to implement these features.
